libc/test/src/math/smoke/CMakeLists.txt

```diff
@@ -92,6 +92,19 @@ add_fp_unittest(
     libc.src.__support.FPUtil.fp_bits
 )
 
+add_fp_unittest(
+  fabsf16_test
+  SUITE
+    libc-math-smoke-tests
+  SRCS
+    fabsf16_test.cpp
+  HDRS
+    FAbsTest.h
+  DEPENDS
+    libc.src.math.fabsf16
+    libc.src.__support.FPUtil.fp_bits
```

Comment:
Copied from the surrounding test target definitions, but I don't see why `fp_bits` is being added as a dependency for `fabs*` tests.

Reply:
Feel free to remove the dependency if `FAbsTest.h` does not use it.